sklearn/metrics/_plot/tests/test_det_curve_display.py

```diff
@@ -56,8 +56,8 @@ def test_det_curve_display(
         pos_label=pos_label,
     )
 
-    assert_allclose(disp.fpr, fpr)
-    assert_allclose(disp.fnr, fnr)
+    assert_allclose(disp.fpr, fpr, rtol=0, atol=1e-7)
+    assert_allclose(disp.fnr, fnr, rtol=0, atol=1e-7)
```

Comment:
Ideally we would not need to change this if we could use an eps value closer to machine precision level.

Reply:
We would still have infinite max relative difference:
```python-traceback
E           AssertionError: 
E           Not equal to tolerance rtol=1e-07, atol=0
E           
E           Mismatched elements: 1 / 1 (100%)
E           Max absolute difference: 2.22044605e-16
E           Max relative difference: inf
E            x: array([2.220446e-16])
E            y: array([0.])

```